lesson_12/homework_12.py

```diff
@@ -1,10 +1,31 @@
+from pathlib import Path
+import json
+import logging
+import xml.etree.ElementTree as ET
+
 # task 1
 """ Візміть два файли з теки
 ideas_for_test/work_with_csv
 порівняйте на наявність дублікатів
 результат запишіть у файл result_<your_second_name>.csv
 """
 
+path_random_csv = Path('C:/Python study Slavych/automation_qa/ideas_for_test/work_with_csv/random.csv')
+path_random_michaels_csv = Path('C:/Python study Slavych/automation_qa/ideas_for_test/work_with_csv/random-michaels.csv')
+result_Slavych_csv = Path('C:/Python study Slavych/automation_qa/ideas_for_test/work_with_csv/result_Slavych.csv')
+
+uniq_elements = set()
+
+with open(path_random_csv, 'r') as file1:
+    for i in file1:
+        uniq_elements.add(i)
+        # print(i)
+
+with open(path_random_michaels_csv, 'r') as file2:
+    with open(result_Slavych_csv, 'w') as result:
+        for a in file2:
+            if a in uniq_elements:
+                result.write(a)
```

Comment:
```suggestion
                result.write(a)
```
тут ви навпаки пишете дублікати